### PR TITLE
Updates, updates, updates

### PR DESCRIPTION
**Pull request by wendigo**

### CODE REVIEW

**Comment by kokosing** [comment]
CI is red

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <parent>
```
Comment (kokosing) [inline pom.xml:5]: maybe you can sort in separate commit, that way it would be easier to follow changes

**Review by kokosing — APPROVED** [review]
% comment

File: pom.xml (hunk 1 above)
Comment (wendigo) [reply pom.xml:5]: Done.

**Pull request merged**